parser/preprocessed/SqlLexer_gen.cpp

```diff
@@ -8,8 +8,8 @@
 
 #define FLEX_SCANNER
 #define YY_FLEX_MAJOR_VERSION 2
-#define YY_FLEX_MINOR_VERSION 6
-#define YY_FLEX_SUBMINOR_VERSION 0
+#define YY_FLEX_MINOR_VERSION 5
```

Comment:
The master branch lists 2.5.38 as  the minimum requirement for the FLEX version. So any version > than that should  be fine. However, there is a TODO  
 # TODO(chasseur): This might not be the exact version to test for.
  if (FLEX_VERSION VERSION_GREATER 2.5.38)
We might need to get this fixed.